code/game/objects/items/melee/misc.dm

```diff
@@ -139,7 +139,7 @@
 	if(!QDELETED(affecting) && !(affecting.bodypart_flags & BODYPART_UNREMOVABLE) && affecting.owner == user && !QDELETED(user))
 		playsound(user, hitsound, 25, TRUE)
 		affecting.dismember(BRUTE)
-		user.adjustBruteLoss(20)
+		user.adjustBruteLoss(23)
```

Comment:
honestly i dont like this magic number being here, it needs to be relative to force

Reply:
ping me on discord after cuz ill be good to merge